### ISSUE TITLE
Things that I could add to increase accuracy

### ISSUE THREAD

**Issue by willyg23** [willyg23/UFC_Machine_Learning_Predicition#3]
mango advice on what to add: 
-RELU and softmax activation methods
-functional APIis for more complex things. multiple inputs mapped to multiple outputs
-sequential APIs less complex, one input to one output

Adeife advice on what to add:
-you could add batch normalization, max pulling, convolution neural networks use many layers to predict output
